x/concentrated-liquidity/lp_test.go

```diff
@@ -24,42 +24,52 @@ type lpTest struct {
 }
 
 var (
-	denom0   = "eth"
-	denom1   = "usdc"
 	baseCase = &lpTest{
 		poolId:          1,
-		currentTick:     sdk.NewInt(85176),
-		lowerTick:       int64(84222),
-		upperTick:       int64(86129),
-		currentSqrtP:    sdk.MustNewDecFromStr("70.710678118654752440"), // 5000
-		amount0Desired:  sdk.NewInt(1000000),                            // 1 eth
+		currentTick:     DefaultCurrTick,
+		lowerTick:       DefaultLowerTick,
+		upperTick:       DefaultUpperTick,
+		currentSqrtP:    DefaultCurrSqrtPrice,
+		amount0Desired:  DefaultAmt0,
 		amount0Minimum:  sdk.ZeroInt(),
-		amount0Expected: sdk.NewInt(998587),     // 0.998587 eth
-		amount1Desired:  sdk.NewInt(5000000000), // 5000 usdc
+		amount0Expected: DefaultAmt0Expected,
+		amount1Desired:  DefaultAmt1,
 		amount1Minimum:  sdk.ZeroInt(),
-		amount1Expected: sdk.NewInt(5000000000), // 5000 usdc
-		liquidityAmount: sdk.MustNewDecFromStr("1517818840.967515822610790519"),
+		amount1Expected: DefaultAmt1Expected,
+		liquidityAmount: DefaultLiquidityAmt,
 	}
 )
 
 func (s *KeeperTestSuite) TestCreatePosition() {
 	tests := map[string]lpTest{
-		// "base case": *baseCase,
-		"amount of token 0 is smaller than minimum; should fail and not update state": {
+		"base case": {},
+		"error: non-existent pool": {
```

Comment:
Do you mind checking if it makes sense to port these edge case tests for `withdrawPosition` as well? They should be very similar